backend/src/SlowNews/Config.hs

```diff
@@ -1,21 +1,22 @@
-{-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE DeriveGeneric     #-}
+{-# LANGUAGE OverloadedStrings #-}
 
 module SlowNews.Config where
 
-import           GHC.Generics
-import           Data.Maybe (fromJust)
-import           Data.Aeson   (decode, FromJSON(..))
+import           Data.Aeson           (FromJSON (..), decode)
 import qualified Data.ByteString.Lazy as B
+import           Data.Maybe           (fromJust)
+import           GHC.Generics
 
 data Config =
     Config { sites :: [Site] }
     deriving (Show, Eq, Generic)
 
 data Site
     = Reddit { subReddit :: String
-             , count :: Maybe Int }
-    | HackerNews
+             , count     :: Maybe Int }
+    | HackerNews { query :: Maybe String
+                 , count :: Maybe Int }
```

Comment:
Sums with records can be dangerous. For example, the following will blow up afaik

``` haskell
foo :: Site -> String
foo = subReddit

foo $ HackerNews (Just "uhoh") (Just 1)
```

Reply:
`-Wincomplete-record-updates` might help here - see https://functor.tokyo/blog/2017-07-28-ghc-warnings-you-should-enable

Reply:
@FintanH Done here: https://github.com/srid/slownews/pull/13/commits/c71af7ca8c172696cc5ff464e35b639efd2965e2